contracts/ICO_crowdsale.sol

```diff
@@ -105,9 +105,10 @@ contract WhitelistedCrowdsale is Ownable, ERC223Receiver {
     }
 
     // Override this method to have a way to add business logic to your crowdsale when buying
-    function getTokenAmount(uint256 weiAmount) internal view returns (uint256) {
+    function getTokenAmount(uint256 weiAmount) public view returns (uint256) {
```

Comment:
@Lontkovsky if you need to change any of the contract code for tests, please discuss it first.
It actually would be better to keep it internal and in tests you could call buyTokens and verify the amount of tokens bought.

Also, in this particular case, it doesn't seem to be a big issue. @Norfolks what do you think? Can we keep the `getTokenAmount` public or it is better to change it back to `internal`?